src/tests/unit/tests/DetailsView/components/overview-content/overview-content-container.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('OverviewContainer', () => {
         actionInitiators: {
             openExternalLink,
         },
-    } as HelpLinkDeps;
+    } as OverviewHelpSectionDeps;
```

Comment:
The name of the variable echoes the name of the type here; should it be updated alongside the type update?

Reply:
Make sense.